tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -382,6 +381,13 @@ func (iptConfigurator *IptablesConfigurator) run() {
 	iptConfigurator.iptables.AppendRuleV4(constants.ISTIOINBOUND, constants.NAT, "-p", constants.TCP, "--dport",
 		iptConfigurator.cfg.InboundTunnelPort, "-j", constants.RETURN)
 
+	if redirectDNS {
+		// redirect all TCP dns traffic on port 53 to the agent on port 15053
+		iptConfigurator.iptables.AppendRuleV4(
+			constants.ISTIOREDIRECT, constants.NAT, "-p", constants.TCP, "--dport", "53", "-j", constants.REDIRECT, "--to-ports", constants.IstioAgentDNSListenerPort)
```

Comment:
Are there any valid reasons to have non dns traffic over port 53? I assume no but figured I would ask..

Reply:
i hope not. if people do, i have no clue to how sort out that mess..